cmake/KatanaConfig.cmake.in

```diff
@@ -35,7 +35,6 @@ if (@NUMA_FOUND@)
   include("${KATANA_CMAKE_DIR}/FindNUMA.cmake")
 endif()
 
-find_dependency(AWSSDK REQUIRED COMPONENTS transfer)
```

Comment:
Why are you removing this? If it's not required by the open repo at all we should remove it more widely.

Reply:
The aws SDK doesn't appear prominently anywhere else:

```shell
ag aws
docs/env.md
6:- `AWS_DEFAULT_REGION`: If set and no region has been otherwise configured
7:  (e.g., via $HOME/.aws/config), use this as the AWS region for accessing
9:  configuration behavior of the AWS S3 CLI client.
10:- `KATANA_AWS_TEST_ENDPOINT`: If set, use this as the endpoint to access S3
11:  rather than the standard AWS endpoint(s). This can be useful for testing.

conda_recipe/environment.yml
8: - awscli

inputs/CMakeLists.txt
3:set(INPUT_URL "https://katana-ci-public.s3.us-east-1.amazonaws.com/inputs/katana-inputs-${INPUT_VERSION}.tar.gz")

CONTRIBUTING.md
435:  https://katana-ci-public.s3.us-east-1.amazonaws.com/inputs/katana-inputs-vN.tar.gz

libgalois/include/katana/SharedMemSys.h
16: * downstream implementation dependencies like the AWS SDK cannot be

python/katana/example_utils.py
32:        "https://katana-ci-public.s3.us-east-1.amazonaws.com/inputs/katana-inputs-v19.tar.gz"

libtsuba/include/tsuba/Errors.h
22:  AWSWrongRegion = 9,
60:    case ErrorCode::AWSWrongRegion:
61:      return "AWS op may succeed in other region";
95:    case ErrorCode::AWSWrongRegion:
```
I can remove reference to awscli in the dev environment (conda/environment.yml) if you want.

Reply:
OK. My bad. I thought it was in the deps elsewhere. Carry on.